framework/web/JsonParser.php

```diff
@@ -18,7 +18,7 @@
  * ```php
  * 'request' => [
  *     'parsers' => [
- *         'application/json' => 'yii\web\JsonParser',
+ *         'application/json' => yii\web\JsonParser::class,
```

Comment:
Please rollback this change.

Reply:
@machour why you want save legacy style? it's not useful now.

Reply:
1. This change have absolutely nothing to do with your PR.
2. Examples are supposed to be compatible with PHP 5.4, as is the framework.